src/app/components/Metadata/index.jsx

```diff
@@ -43,6 +43,38 @@ const getMetaTags = (metaTags, showArticleTags) => {
   ));
 };
 
+const getAppleTouchIconLinks = service => {
+  const sizes = [
+    '72x72',
+    '96x96',
+    '128x128',
+    '144x144',
+    '152x152',
+    '192x192',
+    '384x384',
+    '512x512',
+  ];
+  return sizes.map(size => (
+    <link
+      rel="apple-touch-icon"
+      sizes={size}
+      href={`https://news.files.bbci.co.uk/include/articles/public/${service}/images/icons/icon-${size}.png`}
+    />
+  ));
+};
+
+const getIcons = service => {
```

Comment:
```suggestion
const getIconLinks = service => {
```

Reply:
Switched to using iconAssetUrl()

Reply:
I think that we could still use your first approach `getIconLinks`, but calling the `iconAssetUrl` from inside of it.

Reply:
👍 Changed up my first approach though